src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -182,8 +184,9 @@ def create_project_uptime_subscription(
     Creates an UptimeSubscription and associated ProjectUptimeSubscription
     """
     if mode == ProjectUptimeSubscriptionMode.MANUAL:
-        manual_subscription_count = ProjectUptimeSubscription.objects.filter(
-            project__organization=project.organization, mode=ProjectUptimeSubscriptionMode.MANUAL
+        manual_subscription_count = Detector.objects.filter(
+            project__organization=project.organization,
+            config__mode=ProjectUptimeSubscriptionMode.MANUAL,
```

Comment:
This should also filter to type. Then, since most uptime monitors in a project are MANUAL, the json query part should be fine.